app/models/exceptions/exam_gone_exception.rb

```diff
@@ -0,0 +1,2 @@
+class Exceptions::ExamGoneException < StandardError
```

Comment:
I think that ruby convention is naming exceptions with `Error` suffix instead of `Exception` suffix